src/tests/flyer.test.ts

```diff
@@ -0,0 +1,168 @@
+
+    const getFlyersResponse = await FlyerRepo.getFlyerByID(id);
+    expect(getFlyersResponse.title).toEqual(flyers[0].title);
+  });
+  test('getFlyersByIDs - 3 flyers', async () => {
+    const flyers = await FlyerFactory.create(3);
+    const insertOutput = await FlyerModel.insertMany(flyers);
+    const ids = FactoryUtils.mapToValue(insertOutput, '_id');
+    const getFlyersResponse = await FlyerRepo.getFlyersByIDs(ids);
+
+    expect(FactoryUtils.mapToValue(getFlyersResponse, 'title')).toEqual(
+      FactoryUtils.mapToValue(flyers, 'title'),
+    );
+  });
+});
+
+describe('getFlyersByOrganizationSlug(s) tests', () => {
+  test('getFlyersByOrganizationSlug - 1 organization, 1 flyer', async () => {
+    const pub = await OrganizationFactory.getRandomOrganization();
+    const flyers = await FlyerFactory.createSpecific(1, {
+      organizationSlug: pub.slug,
+      organization: pub,
+    });
+    await FlyerModel.insertMany(flyers);
+
+    const getFlyersResponse = await FlyerRepo.getFlyersByOrganizationSlug(pub.slug);
+    expect(getFlyersResponse[0].title).toEqual(flyers[0].title);
+  });
+
+  test('getFlyersByOrganizationSlug - 1 organization, 3 flyers', async () => {
+    const pub = await OrganizationFactory.getRandomOrganization();
+    const flyers = (
+      await FlyerFactory.createSpecific(3, {
+        organizationSlug: pub.slug,
+        organization: pub,
+      })
+    ).sort(FactoryUtils.compareByDate);
+
+    await FlyerModel.insertMany(flyers);
+    const getFlyersResponse = await FlyerRepo.getFlyersByOrganizationSlug(pub.slug);
+
+    expect(FactoryUtils.mapToValue(getFlyersResponse, 'title')).toEqual(
+      FactoryUtils.mapToValue(flyers, 'title'),
+    );
+  });
+
+  test('getFlyersByOrganizationSlugs - many organizations, 5 flyers', async () => {
+    const flyers = (await FlyerFactory.create(3)).sort(FactoryUtils.compareByDate);
+
+    await FlyerModel.insertMany(flyers);
+    const getFlyersResponse = await FlyerRepo.getFlyersByOrganizationSlug(
+      FactoryUtils.mapToValue(flyers, 'organizationSlug'),
+    );
+
+    expect(FactoryUtils.mapToValue(getFlyersResponse, 'title')).toEqual(
+      FactoryUtils.mapToValue(flyers, 'title'),
+    );
+  });
+});
+
+describe('getFlyersAfterDate tests', () => {
+  test('getFlyersAfterDate - filter 1 flyer', async () => {
+    const today = new Date();
+    let flyers = await FlyerFactory.createSpecific(1, {
+      date: faker.date.recent(1), // 1 day ago
+    });
+    flyers = flyers.concat(
+      await FlyerFactory.createSpecific(1, {
+        date: faker.date.past(1, today.getDate() - 2), // 1 year ago
+      }),
+    );
+    await FlyerModel.insertMany(flyers);
+
+    const getFlyersResponse = await FlyerRepo.getFlyersAfterDate(
+      faker.date.recent(2, today.getDate() - 1).toString(),
+    ); // find flyers from after 2 days ago
+    expect(getFlyersResponse[0].title).toEqual(flyers[0].title);
+  });
+});
+
+describe('searchFlyer tests', () => {
```

Comment:
add test for limit and a test where you expect 0 results

Reply:
will do! thanks for the suggestion